content/docs/command-reference/queue/start.md

```diff
@@ -0,0 +1,61 @@
+## queue start
+
+Start the
+[DVC experiments](/doc/user-guide/experiment-management/experiments-overview)
+task queue worker process.
+
+## Synopsis
+
+```usage
+usage: dvc queue start [-h] [-q | -v] [-j <number>]
+```
+
+## Description
+
+Starts one or more task queue worker processes. Each worker process will consume
+and execute one queued experiment task at a time in the background, until either
+`dvc queue stop` is used or the queue is empty.
+
+<admon type="info">
+
+Due to [internal limitations], when the queue is empty a worker
+may be idle for up to 10 seconds before exiting. If new
+experiment tasks are added to the queue during this time, the idle worker will
+resume processing them instead.
+
+[implementation limitations]:
+  /doc/user-guide/experiment-management/running-experiments#how-are-experiments-queued
+
+</admon>
+
+Queued experiment tasks are run sequentially by default, but can be run in parallel by using
+the `--jobs` option to start  more than one worker.
+
+> ⚠️ Parallel runs are experimental and may be unstable. Make sure you're using
+> a number of jobs that your environment can handle (no more than the CPU
+> cores).
```

Comment:
Please use `<admon>` instead of block quotes throughout. See https://github.com/iterative/cml.dev/pull/210#issue-1178579878 for usage info. Thanks